### PR TITLE
💄 Logging for image size

### PR DESCRIPTION
**Pull request by aileen**
no issue

- added debug logs to image size util and related fn:
    - when fetched via network request
    - when fetched from storage
    - when added to cache
    - when read from cache

### CODE REVIEW

File: core/server/adapters/storage/utils.js [hunk 1]
```diff
@@ -1,27 +1,47 @@
...
     }
 };
+
+/**
+ * @description compares the imagePath with a regex that reflects our local file storage
+ * @param {String} imagePath as URL or filepath
+ * @returns {Boolean}
+ */
+
+exports.isLocalImage = function isLocalImage(imagePath) {
```
Comment (kirrg001) [inline core/server/adapters/storage/utils.js]: This function is basically doing what `getLocalFileStoragePath` is doing, except of the return value 🤔 
As these conditions are quite long and it takes a bit to read them, how about doing

```
x = getLocalFileStoragePath(path)
if (x !== path) {
   return true;
}

return false;
```

**Comment by kirrg001** [comment]
FYI: An empty file was added -> `core/test/unit/utils/request_spec.js`.

File: core/server/adapters/storage/utils.js (hunk 1 above)
Comment (AileenCGN) [reply core/server/adapters/storage/utils.js]: Thank you for this! Totally makes sense to me and looks sooo much better now!

**Comment by AileenCGN** [comment]
@kirrg001 Please don't merge this yet. I'm doing further investigations and will update this PR in a bit.

**Comment by AileenCGN** [comment]
@ErisDS This is ready to go!

**Pull request merged**